prep-javascript-objects/main.js

```diff
@@ -0,0 +1,20 @@
+
+var person = {
+
+  firstName: 'Andre',
+  lastName: 'Le',
+  hobby: 'Photography'
+};
+console.log(person);
+
+var fullName = ('Andre' + ' ' + 'Le');
+
```

Comment:
You can access the person object's `firstName` and `lastName` properties with dot notation.

```js
var fullName = person.firstName + ' ' + person.lastName;
```

Reply:
I have updated the following: 
![image](https://user-images.githubusercontent.com/103016667/166625299-994584bd-9e8f-472b-8f0c-d48daeeb4464.png)